packages/sequelize/src/adapter/index.js

```diff
@@ -0,0 +1,416 @@
+import "core-js/stable";
+import "regenerator-runtime/runtime";
+import { createHash, randomBytes } from "crypto";
+
+import { CreateUserError } from "./lib/errors";
```

Comment:
```suggestion
import { CreateUserError } from "next-auth/errors";
```

These are included in the base library now. check out the content of the errors module, there are more type of errors as well. You can delete the `./lib/errors` file now